### PR TITLE
OSError

### PR DESCRIPTION
**Pull request by thisiswei**
One of my cron command was failing but it was not caught by cron-sentry

Issue #3

### CODE REVIEW

File: cron_sentry/runner.py [hunk 1]
```diff
@@ -119,7 +119,10 @@ def run(self):
 
         with TemporaryFile() as stdout:
             with TemporaryFile() as stderr:
-                exit_status = call(self.command, stdout=stdout, stderr=stderr)
+                try:
+                    exit_status = call(self.command, stdout=stdout, stderr=stderr)
+                except OSError:
+                    exit_status = 1
 
                 last_lines_stdout = self._get_last_lines(stdout)
```
Comment (hltbra) [inline cron_sentry/runner.py]: I think this will be a problem when you have OSError.

File: cron_sentry/runner.py [hunk 2]
```diff
@@ -119,7 +119,10 @@ def run(self):
 
         with TemporaryFile() as stdout:
             with TemporaryFile() as stderr:
-                exit_status = call(self.command, stdout=stdout, stderr=stderr)
+                try:
+                    exit_status = call(self.command, stdout=stdout, stderr=stderr)
+                except OSError:
+                    exit_status = 1
```
Comment (hltbra) [inline cron_sentry/runner.py]: Maybe it's a better idea to report the exception to Sentry right here.

File: cron_sentry/runner.py (hunk 1 above)
Comment (thisiswei) [inline cron_sentry/runner.py]: Sorry I don't follow, what scenarios are you thinking?

**Comment by thisiswei** [comment]
try using following code but the result were the same:

```
    def run(self):
        start = time()

        with TemporaryFile() as stdout:
            with TemporaryFile() as stderr:
                try:
                    exit_status = call(self.command, stdout=stdout, stderr=stderr)
                except OSError as exc:
                    exit_status = 1
                    elapsed = int((time() - start) * 1000)
                    self.report_fail(1, '', unicode(exc), elapsed)
                    return exit_status

                last_lines_stdout = self._get_last_lines(stdout)
                last_lines_stderr = self._get_last_lines(stderr)

                if exit_status > 0:
                    elapsed = int((time() - start) * 1000)
                    self.report_fail(exit_status, last_lines_stdout, last_lines_stderr, elapsed)

                if not self.quiet:
                    sys.stdout.write(last_lines_stdout)
                    sys.stderr.write(last_lines_stderr)

                return exit_status
```


this is after lines added: https://sentry.io/yipitcom/tests/issues/190856199/
this was before: https://sentry.io/yipitcom/tests/issues/190853717/

**Pull request merged**